docs/resources/rule_parser_state.md

```diff
@@ -29,53 +29,53 @@ resource "wallarm_rule_parser_state" "disable_xml_parsing" {
 
 ## Argument Reference
 
-* `client_id` - (Optional) ID of the client to apply the rules to. The value is required for multi-tenant scenarios.
-* `action` - (Optional) Rule conditions. Possible attributes are described below.
-* `parser` - (Required) Parser to enable/disable. Possible values: `base64`, `cookie`, `form_urlencoded`, `gzip`, `grpc`, `json_doc`, `multipart`, `percent`, `protobuf`, `htmljs`, `viewstate`, `xml`.
-* `state` - (Required) Desired state for the parser. Possible values: `enabled`, `disabled`.
-* `point` - (Required) Request parts to apply the rules to. The full list of possible values is available in the [Wallarm official documentation](https://docs.wallarm.com/user-guides/rules/request-processing/#identifying-and-parsing-the-request-parts).
+* `client_id` - (optional) ID of the client to apply the rules to. The value is required for [multi-tenant scenarios][2].
+* `action` - (optional) rule conditions. Possible attributes are described below.
+* `parser` - (**required**) parser to enable/disable. Possible values: `base64`, `cookie`, `form_urlencoded`, `gzip`, `grpc`, `json_doc`, `multipart`, `percent`, `protobuf`, `htmljs`, `viewstate`, `xml`.
```

Comment:
there are more parsers, please check in other docs as well

Reply:
Only these are currently supported by the provider.